8-toolchain/Dockerfile.fedora

```diff
@@ -0,0 +1,45 @@
+FROM registry.fedoraproject.org/f27/s2i-core:latest
```

Comment:
Just a question. Do we really want to support this a bit older image?
What about at least f29? Or even f31 where is a fixed issue https://github.com/sclorg/s2i-base-container/pull/198 although it is not related to C/C++.